.github/workflows/tox.yml

```diff
@@ -18,6 +18,15 @@ jobs:
       - uses: actions/setup-python@v2
         with:
           python-version: ${{ matrix.python-version }}
+      - name: Cache multiple paths
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.cache/pip
+            /opt/hostedtoolcache/Python/
+            ~\AppData\Local\pip\Cache
+            c:\hostedtoolcache\windows\python\
+          key: ${{ runner.os }}-build-${{ env.cache-name }}
```

Comment:
`env.cache-name` should be `matrix.python-version`, or we might end up with the wrong wheels, for python version sensitive packages.

Reply:
Good point! I will amend this across all of the workflows.